lib/vast/Target/LLVMIR/Convert.cpp

```diff
@@ -30,6 +30,52 @@ VAST_UNRELAX_WARNINGS
 
 namespace vast::target::llvmir
 {
+    namespace
+    {
+        // TODO(target): Unify with tower and opt.
+        void populate_pm(mlir::PassManager &pm, pipeline p)
+        {
+            switch (p)
+            {
+                case pipeline::baseline:
+                {
+                    pm.addPass(hl::createHLLowerTypesPass());
```

Comment:
group shared passes between pipelines so we don't repeat ourself:
   - simplify-hl: `HLLowerTypes`, `DCE`, `LowerTypeDefs`
   - abi
   - to-ll
   - to-llvm
 and let `baseline` and `with_abi` invoke these pipelines?